### PR TITLE
.value --> .get()

### PR DESCRIPTION
**Pull request by mrakitin**
Observed during SRX deployment (Part 3) (https://github.com/NSLS-II-SRX/profile_collection/issues/10).

### CODE REVIEW

**Comment by mrakitin** [comment]
Thanks for reviewing this, @danielballan! Do you mind to check https://github.com/NSLS-II-HXN/hxntools/pull/19 as well? Identical updates.

File: nslsii/iocs/thermo_sim.py [hunk 1]
```diff
@@ -52,7 +52,7 @@ def t_rbv(T0, setpoint, K, omega, Tvar,):
                     setpoint)
 
         T = t_rbv(T0=self._T0,
-                  **{k: getattr(self, k).value
```
Comment (tacaswell) [inline nslsii/iocs/thermo_sim.py:55]: This is inside of a caproto IOC and I don't think will work!

Comment (mrakitin) [reply nslsii/iocs/thermo_sim.py]: Reverted this change. Thanks for catching it up!

**Comment by mrakitin** [comment]
@tacaswell, are you fine with the changes now? Thanks.

**Pull request merged**